zerver/tests/test_events.py

```diff
@@ -1763,3 +1779,4 @@ def test_use_narrow_if_default_is_none(self):
         self.user_profile.save()
         result = _default_narrow(self.user_profile, [])
         self.assertEqual(result, [])
+
```

Comment:
Aside from this trailing whitespace change being unnecessary (you should amend to remove it), the code looks good!